layers/meta-resin-allwinner/recipes-kernel/rtl8189fs/rtl8189fs/0001-Disable-debug-messages.patch

```diff
@@ -0,0 +1,25 @@
+From f2513e71dffdb207a7d71f7c52bac0f67b9fbcb4 Mon Sep 17 00:00:00 2001
+From: Michel Wohlert <michel.wohlert@googlemail.com>
+Date: Mon, 4 Mar 2019 08:18:53 +0100
+Subject: [PATCH] Disable debug messages
+
+---
```

Comment:
Please include `Upstream-status` and `Signed-off-by`

Reply:
Done